### PR TITLE
Add an admin smite for making people slip really far (and localize the admin smites better)

### PR DESCRIPTION
**Pull request by notquitehadouken**
## About the PR

added an admin smite that sends someone slipping really far

## Why / Balance

an admin asked for it
funni

## Media

https://github.com/space-wizards/space-station-14/assets/45953835/2fc1ffac-6267-42d9-a07d-af3c89baa3b9

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

Made the TrySlip method in SlipperySystem public

**Changelog**

no cl no fun

### CODE REVIEW

**Comment by Jarmer123** [comment]
I in fact did ask for this :godo:

**Comment by MerrytheManokit** [comment]
> I in fact did ask for this :godo:

Of course it was you lol

**Comment by QuietlyWhisper** [comment]
Doesn't slip far enough 6/10

**Comment by SoulFN** [comment]
I think it should slip you until you hit the wall or something like this

**Comment by notquitehadouken** [comment]
> Doesn't slip far enough 6/10

fine i'll multiply the slip speed by 60

**Comment by notquitehadouken** [comment]
> I think it should slip you until you hit the wall or something like this

but that's hard

**Comment by crazybrain23** [comment]
Won't they clip through walls?

**Comment by deathride58** [comment]
> Won't they clip through walls?

Feature

**Comment by notquitehadouken** [comment]
> How much damage does it do if you get slipped near a wall? iirc the slipforwardsmodifier just sets your speed, and going too fast can slam you into things so fast you gib.

idfk it's an admin smite, just repeat until greytide shittery ceases

**Comment by Scruq445** [comment]
> > Won't they clip through walls?
> 
> Feature

But that's already an admin smite

File: Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs [hunk 1]
```diff
@@ -846,5 +848,31 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
             Message = Loc.GetString("admin-smite-terminate-description")
         };
         args.Verbs.Add(terminate);
+
+        Verb superslip = new()
+        {
+            Text = "Super Slip",
```
Comment (metalgearsloth) [inline Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs:854]: Needs localisation.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs (hunk 1 above)
Comment (notquitehadouken) [reply Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs:854]: so you want me to add localisation for every single admin smite's text?

Comment (notquitehadouken) [reply Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs:854]: > I assume just this one unless you want to go through and do all the others while adding it here

sure why not

**Comment by notquitehadouken** [comment]
OH GOD

**Pull request merged**